src/main/java/core/basesyntax/animals/Aviaries.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.animals;
+
+import core.basesyntax.kind.Birds;
+import core.basesyntax.kind.Fishes;
+import core.basesyntax.kind.LandAnimal;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Aviaries {
+    public static List<LandAnimal> landAnimalAviary = new ArrayList<>();
+    public static List<Fishes> fishesAquarium = new ArrayList<>();
+    public static List<Birds> penguinAquarium = new ArrayList<>();
+    public static List<Birds> birdsAviary = new ArrayList<>();
```

Comment:
it is better to create a separate class for each animal "container"
use generics to indicate which animals can be put in each container